### PR TITLE
feat: PUT/PATCH/DELETE /decrees/{decree_id} with FGA gating

### PR DESCRIPTION
**Pull request by JohnRDOrazio**
Implements the API half of `docs/superpowers/specs/2026-07-11-decrees-write-paths-design.md`. Refs #706 (debuts the per-item creation path shape).

## Summary

- **`PUT /decrees/{decree_id}`** — create (201; 409 on existing id). Requires FGA `editor` on `general_roman_calendar:decrees`.
- **`PATCH /decrees/{decree_id}`** — update (200; 404 unknown; 400 if the payload tries to change `liturgical_event.event_key` — that requires DELETE + PUT). Requires `editor`.
- **`DELETE /decrees/{decree_id}`** — delete (200; 404 unknown) with garbage collection of the decree's `event_key` from `i18n/*.json` and `lectionary/*.json` only when no surviving decree references it. Requires `admin`.
- `GET /decrees(/{id})` responses now include `liturgical_event.readings` (from `decrees/lectionary/{locale}.json`, base-locale fallback, property absent when no readings exist). GET remains public.

## Authorization

Existing pipeline (`JwtAuthMiddleware` → `AuthorizationMiddleware::forCalendarEditor()` → `OpenFgaAuthorizationMiddleware::forGeneralRomanCalendar`) with a decrees-specific relation map (`PUT/PATCH→editor`, `DELETE→admin` — the tests-endpoint map). `forGeneralRomanCalendar()` gained an optional relation-map parameter; temporale's call site and behavior are unchanged. Global admins bypass FGA as everywhere. Permission granting stays on `PermissionAdminHandler` (grc resource admins can already grant there).

## Payload validation (three layers)

1. New JSON schema `LitCalDecreeWritePayload.json` (`LitSchema::DECREE_WRITE`, registered in the schema corpus test): single decree + write-only `i18n`/`readings` sidecars.
2. `DecreeWritePayloadGuard`: per-action sidecar matrix (i18n required for `createNew`/`makeDoctor`/`setProperty:name` incl. the Accept-Language base locale, rejected for `setProperty:grade`; readings required on PUT only for `createNew`, optional on PATCH) + `LitLocale` validation of sidecar locale keys.
3. `DecreeItem` DTO invariants (fixed vs mobile shapes, etc.), validated on a deep copy so validation can never mutate stored data.

## Data handling

Single-database writes to `decrees.json` (append / in-place replace / remove) with `JsonFormatter` + `LOCK_EX`; i18n distributed to every existing locale file (provided values win, existing translations preserved, empty-string placeholder for new keys — Weblate fills later); readings distributed to provided locales only; sidecar write failures surface as 503; audit-log entries (operation, decree_id, user, ip, files) on every write.

## Tests

- 14 new/extended in-process handler tests (hermetic: snapshot/restore of the decrees data folder)
- 8 pure guard tests, 4 schema tests, middleware relation-map test
- `Routes/ReadWrite/DecreesTest` (401/409/400/404 + full PUT→PATCH→DELETE lifecycle with cleanup); write cases skip against a stale local server, strict in CI
- Full quick suite: 1528 tests / 153215 assertions green; PHPStan level 10 clean; `composer lint:openapi` valid

## Review

Subagent-driven: per-task spec+quality reviews, then a whole-branch final review whose findings (incl. a shallow-clone data-corruption bug) were all fixed in bad18646 and re-verified empirically.

Frontend companion PR (admin-decrees page) follows separately per the spec.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **New Features**
  * Added authenticated, permission-gated `PUT/PATCH/DELETE /decrees/{decree_id}` with locale sidecar rules and robust rollback behavior.
  * `GET /decrees/{decree_id}` is now enriched with aggregated `i18n` and locale `readings` (including fallback).
  * Added Temporale (“Proprium de Tempore”) entries to `/events`.
* **Documentation**
  * Updated OpenAPI for decree write operations and request/response payloads.
* **Bug Fixes**
  * Improved admin permission checks with a self-check exemption.
* **Tests**
  * Expanded schema, route, authorization, and edge-case coverage (including rollback/orphan cleanup).

### CODE REVIEW

File: src/Router.php [hunk 1]
```diff
@@ -716,7 +716,11 @@ private function configureAuthorizationPipeline(
         } elseif ($route === 'decrees') {
             $pipeline->pipe(AuthorizationMiddleware::forCalendarEditor());
             if ($oidcAvailable && $fgaClient !== null) {
-                $pipeline->pipe(OpenFgaAuthorizationMiddleware::forGeneralRomanCalendar($fgaClient, 'decrees'));
+                $pipeline->pipe(OpenFgaAuthorizationMiddleware::forGeneralRomanCalendar(
+                    $fgaClient,
+                    'decrees',
+                    ['PUT' => 'editor', 'PATCH' => 'editor', 'DELETE' => 'admin']
+                ));
             }
```
[the remark this answers is not in the document]
Comment (JohnRDOrazio) [reply src/Router.php:724]: deliberate platform-wide pattern — a decrees-only change would diverge from /temporale//tests//missals
